scripts/dts/extract_dts_includes.py

```diff
@@ -426,8 +426,9 @@ def load_binding_file(fname):
                        .format(fname))
 
     if len(filepaths) > 1:
-        yaml_inc_error("Error: multiple candidates for file name '{}' in "
-                       "!include statement: {}".format(fname, filepaths))
+        print("Warning: multiple candidates for file name "
+              "'{}' in !include statement - using first of".
+              format(fname), filepaths)
```

Comment:
Could use just `.format()` instead of a mix of `.format()` and multiple arguments to `print()`:

```python
print("Warning: multiple candidates for file name "
      "'{}' in !include statement - using first of {}"
      .format(fname, filepaths))
```

Maybe a warning helper could be added somewhere to make the warning go to `stderr`, without cluttering things up with a bunch of `file=sys.stderr` everywhere:

```python
def warn(*args):
    print('warning: ', end='', file=sys.stderr)
    print(*args, file=sys.stderr)
```

Reply:
@ulfalizer 
> Could use just `.format()` instead of a mix of `.format()` and multiple arguments to `print()`
 
Sorry I don´t see multiple arguments - there is **no** comma between the strings.

The wole thing about splitting the string is the Zephyr coding guideline that still insists on a last century rule about 80 chars per line. The time of 80 char/ line terminals and dot-matrix printers.

> Maybe a warning helper could be added

Shure. But should be in another PR.

Reply:
There are two arguments to the `print()` function.

It's like writing

```python
print('foo {}'.format('bar'), 'baz')
```

instead of

```python
print('foo {} {}'.format('bar', 'baz'))
```

It looks confusing.

Reply:
ok, got it. done.